modules/otel-troubleshooting-instrumentation-injection-into-your-workload.adoc

```diff
@@ -0,0 +1,93 @@
+/ Module included in the following assemblies:
```

Comment:
This is showing up verbatim in the build because it's just the one `/`, should be two for a comment:
```suggestion
// Module included in the following assemblies:
```

Reply:
Oops, sorry about this.